doc/devel/gitwash/development_workflow.rst

```diff
@@ -397,7 +397,7 @@ Deleting a branch on GitHub_
    # delete branch on GitHub
    git push origin :my-unwanted-branch
 
-(Note the colon ``:`` before ``test-branch``.  See also:
+Note the colon ``:`` before ``my-unwanted-branch``.  See also:
 http://github.com/guides/remove-a-remote-branch
```

Comment:
[https://github.com/guides/remove-a-remote-branch](https://github.com/guides/remove-a-remote-branch) does not seem to be valid and redirects toward [https://help.github.com/categories/managing-remotes/](https://help.github.com/categories/managing-remotes/). From there, I have chosen the (https) link that sounded related to "removing a remote branch".